source/Octopus.Tentacle.Client/TentacleClientOptions.cs

```diff
@@ -7,6 +7,11 @@ public class TentacleClientOptions
     {
         public RpcRetrySettings RpcRetrySettings { get; }
         
+        /// <summary>
+        /// Disables the use of ScriptServiceV3Alpha, even if it's supported on the Tentacle
+        /// </summary>
+        public bool DisableScriptServiceV3Alpha { get; set; }
```

Comment:
Nit: I think this would be better with a private set (or read-only) and added via the constructor

Reply:
The reason I didn't make it part of the constructor is that it's not "required" for the options to be "valid". 

We do want it to be off by default on only disabled in certain circumstances (in tests + in Server while we work on the k8s agent)